src/DotNetBoilerplate.Api/Organizations/GetOrganizationEndpoint.cs

```diff
@@ -21,7 +21,7 @@ public static void Map(IEndpointRouteBuilder app)
             CancellationToken ct
         )
         {
-            var query = new GetOrganizationQuery();
```

Comment:
Tutaj też widzę że jest to tak na prawdę https://github.com/mpienk102/digital-feedback/pull/13